src/gluonnlp/data/filtering.py

```diff
@@ -186,14 +186,12 @@ def __init__(self, algo='fasttext_compressed', model_path=None):
             fasttext = try_import_fasttext()
             if model_path is None:
                 if algo == 'fasttext':
-                    model_path = download('https://dl.fbaipublicfiles.com/fasttext/'
-                                          'supervised-models/lid.176.bin',
+                    model_path = download('s3://gluonnlp-numpy-data/misc/fasttext/lid.176.bin',
                                           os.path.join(get_model_zoo_home_dir(),
                                                        'fasttext_langid', 'lid.176.bin'),
                                           sha1_hash='e613bda316ecb4f5e1924140eedf81b81c087d9a')
                 elif algo == 'fasttext_compressed':
-                    model_path = download('https://dl.fbaipublicfiles.com/fasttext/'
-                                          'supervised-models/lid.176.ftz',
+                    model_path = download('s3://gluonnlp-numpy-data/misc/fasttext/lid.176.ftz',
```

Comment:
You may use `https://gluonnlp-numpy-data.s3-us-west-2.amazonaws.com/models/fasttext_langid/lid.176.ftz`